examples/kubernetes-ingress/scripts/01-install-etcd.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
```

Comment:
Add logs using `tests/helpers.bash:log` so users can follow what's going on in case they hit an issue while the script runs.

Reply:
Add comments specifying which variables are populated outside of this script and their meaning. See the Bash Style Guide for function comments: https://google.github.io/styleguide/shell.xml#Function_Comments